joint_teapot/utils/joj3.py

```diff
@@ -201,13 +201,22 @@ def generate_title_and_comment(
         comment += "\n"
         for i, result in enumerate(stage["results"]):
             if not single_case:
-                comment += f"### Case {i} - Score: {result['score']}\n"
+                comment += (
+                    f"<details><summary>Case {i} - Score: {result['score']}</summary>\n"
+                )
             if result["comment"].strip() != "":
                 comment += f"{result['comment']}\n"
```

Comment:
Let us collapse the comment on a single case for consistency, as it might be long if a single case diff is failed.